### DISCUSSION TITLE
Representing disputed areas on the map.

### DISCUSSION THREAD

**Discussion by SunilAnandatheertha** [electricitymaps/electricitymaps-contrib#6670] [General]
Hi, firstly great repo. Thanks for making this. This is in reference to the issue: https://github.com/electricitymaps/electricitymaps-contrib/issues/6669

I request anyone reading this to comment upon fully reading the post. Thank you. I present the case below.

I was looking at India (at https://app.electricitymaps.com/map) and noticed that the northern Indian border is not represented. Representing a disputed border as undisputed by not making known, at-least the presence of any such dispute on the map, has the same effect of shifting the goal post of argument to the other party of claim under consideration. I see this is a valid issue in map representation and in any usage/application of the map representation such as this application itself. This issue MUST be addressed by making sure that any disputed areas of map(s) must indicate the border of dispute in dotted or dashed lines as is done in all maps including those by UN. Below is an example taken from an official map by UN:
https://www.un.org/geospatial/sites/www.un.org.geospatial/files/files/documents/2020/Nov/world_map_4170_r19_oct20.pdf

Below is a contradiction I find to the explanation in the issue I raised (link at the beginning):
While at the time when I started this discussion, https://app.electricitymaps.com/map, clearly demarcates Sudan and South Sudan by a continuous line, even though the border between them is shown by a non-continuous line in the UN map in the above link. In their own words, UN states on their map (link above) "Final boundary between the Republic of Sudan and the Republic of South Sudan has not yet been determined".

There is, therefore, an undeniable major flaw in the accuracy of underlying map data used AND also in the way in which is being represented.

There was no matching previous discussion on these lines. Open to discussion on how https://github.com/electricitymaps/ should be addressing this.

**Comment by madsnedergaard** [comment]
I do not want to say that this is not important(!), but as @VIKTORVAV99 pointed out in the issue it would require significant changes to support "un-associated" areas on the map. For the foreseeable future we have other projects that we deem have a larger impact on our end-goal of decarbonizing the world, and as such we cannot prioritise this issue at the moment. There are also a bunch of other areas in the world that are disputed, but does not show up on our map.

As a secondary comment, the goal of our app is to show how electricity is consumed in countries across the world, and not necessarily to have the most representative and accurate view of country borders - this is also why we have chosen to use simplified geometries for countries.

**Reply by SunilAnandatheertha** [reply]
Thanks for the reply here and in the issue.

Where do you pull the map data from?

As people with a clear connect with socio-economic developments in a disputed region, we would greatly benefit from using this application to track decarbonization drive in these regions.
